### ISSUE TITLE
[AnVIL Portal] Create Next.js instance in Portal

### ISSUE THREAD

**Issue by MillenniumFalconMechanic** [clevercanary/anvil-portal#4]
#### Need
We will be incrementally moving the anvilproject.org site from Gatsby to nextjs. We need a nextjs app in the same repository to start committing to.

#### Approach
Create a nextjs app under a `/next` directory from the root. This can be to get started the most basic next app with the default home page.  We will add customizations in a separate ticket and for now we just need the basic app checked in (client side only, no server for this app).

#### Notes
Once this is done PR to https://github.com/anvilproject/anvil-portal

**Comment by NoopDog** [comment]
Complete!
